syntax/pact.vim

```diff
@@ -25,6 +25,7 @@ endif
 " Special forms
 syn keyword pactSyntax bless defun defcap defconst defpact defschema
 syn keyword pactSyntax deftable let let* step step-with-rollback use module
+syn keyword pactSyntax property interface implements
```

Comment:
For more on properties, see [here](https://pact-language.readthedocs.io/en/stable/pact-properties.html). Interfaces can be read about in the latest documentation that will be released next week